libraries/AC_Sprayer/AC_Sprayer.cpp

```diff
@@ -122,6 +123,7 @@ void AC_Sprayer::update()
 
     // exit immediately if the pump function has not been set-up for any servo
     if (!SRV_Channels::function_assigned(SRV_Channel::k_sprayer_pump)) {
+        gcs().send_text(MAV_SEVERITY_WARNING, "SprayerPump is not set for servo function");
```

Comment:
I guess this will send the message to the console at 3hz but only once the sprayer is activated.  It would probably be better as a pre-arm check but even then I'm not sure it's worth it.  The user should be able to figure it out from reading [the wiki](https://ardupilot.org/copter/docs/sprayer.html).